frontend/scripts/get_discourse_posts.py

```diff
@@ -52,19 +54,33 @@ def get_arguments() -> Any:
         args, unknown = parser.parse_known_args()
         return args
 
+    @staticmethod
+    def _get_cache_path(url: str) -> Path:
+        cache_dir = Path(os.environ['HOME']) / "WebCache"
```

Comment:
Need to copy into here whatever MAVProxy does to work out where the user's *stuff* is - 'HOME' is not in Windows env.

Reply:
fixed